packages/sdk/src/controllers/ActionController.ts

```diff
@@ -139,4 +139,24 @@ export class ActionController {
     setTypeError = async (id: string, hasTypeErrors: boolean) => {
         return this.update(id, { hasTypeError: hasTypeErrors });
     };
+
+    /**
+     * This method disables the execution of Actions.
+     * Note that any Action will be ignored and forgotten.
+     * @returns
+     */
+    disableExecution = async () => {
+        const res = await this.#editorAPI;
+        return res.disableActions().then((result) => getEditorResponseData<null>(result));
+    };
+
+    /**
+     * This method enables the execution of Actions.
+     * Note this is enabled by default.
+     * @returns
+     */
+    enableExecution = async () => {
```

Comment:
just disable / enable? speaks for itself no?

Reply:
Disable actioncontroller? Confusing

Reply:
in the end it becomes SDK.action.disable

Reply:
Would love if it were actions, this now indicates 'disable a single action'. Though in this case disable makes sense indeed